scripts/release-all-packages.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source ./scripts/helpers.sh
+
+releasePackages=()
+for dir in packages/*/ ; do
+  if [[ $(json -f "${dir}package.json" releasePackage) == true ]]; then
+    releasePackages+=("$(cut -d'/' -f2 <<<${dir})")
+  fi
+done
+
+for package in "${releasePackages[@]}"
+do
+	echo "Start releasing package $package"
+	yarn release-package $package --auto
```

Comment:
this will run "yarn setup" for every package since its part of "release-package". maybe this could be fixed by separating this two calls